### PR TITLE
Add new message type

### PR DESCRIPTION
**Pull request by vvasianovych**

### CODE REVIEW

File: src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Slf4j
+@Service
+public class FirebaseService {
```
Comment (major13ua) [inline src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java]: How the multitenancy feature implemented for firebase?

File: src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Value("${application.firebase.url}")
+    private String firebaseURL;
+
+    @Value("${application.firebase.token}")
+    private String token;
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java]: better to move to configuration on configuration ms
maybe add microservice configuration like uaa.yml but for this ms

File: src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java (hunk 1 above)
Comment (ishkurko) [reply src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java]: TMF microservices without multitenancy support at this moment. The same situation with the SMPP connection.

File: src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java (hunk 2 above)
Comment (vvasianovych) [reply src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java]: will be provided as separate PR for configuration MS

Comment (vvasianovych) [reply src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java]: configuration is re-implemented

File: src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class FirebaseService {
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/communication/service/FirebaseService.java]: @vvasianovych please add test for FirebaseService class

**Pull request merged**